PalasoUIWindowsForms/ClearShare/LicenseInfo.cs

```diff
@@ -109,13 +149,29 @@ public override string ToString()
 			return "Custom License";
 		}
 
-		public override string GetDescription(string iso639_3LanguageCode)
+		/// <summary>
+		/// Currently, we don't even know the language of custom license strings, so this implementation just gives you the string it has
+		/// </summary>
+		/// <returns></returns>
+		public override string GetDescription(string iso6393LanguageCode)
 		{
 			if (string.IsNullOrEmpty(RightsStatement))
 				return "For permission to reuse, contact the copyright holder.";
```

Comment:
It could make sense not to localize this, since you can't localize a user-supplied rights statement...but OTOH you COULD localize this...